src/demo/components/common/ContextMenu.ts

```diff
@@ -263,16 +266,15 @@ class ContextMenu {
       font-family: "Segoe UI",Arial,sans-serif;
       font-weight: 400;
     }
-    ul{
-      padding: 0 20px;
-    }
-
     li {
       list-style-type: none;
       margin: 0 auto;
       height: 24px;
       cursor: pointer;
     }
+    li span {
+      padding-left: 20px;
+    }
```

Comment:
Not checked out this branch, but are there still style updates pending?

Last time I saw the context menus, was no indication of which items expanded into sub-menus.

Reply:
I'll add that now.